Framework/PythonInterface/mantid/plots/datafunctions.py

```diff
@@ -1047,24 +1047,34 @@ def apply_waterfall_offset_to_errorbars(ax, line, amount_to_move_x, amount_to_mo
             # Shift the errorbars
             for bar_line_col in container[2]:
                 segments = bar_line_col.get_segments()
+                i = 0
                 for point in segments:
                     for row in range(2):
-                        point[row][1] += amount_to_move_y
+                        point[row][1] += amount_to_move_y[i]
                     for column in range(2):
-                        point[column][0] += amount_to_move_x
+                        point[column][0] += amount_to_move_x[i]
+                    i+=1
                 bar_line_col.set_segments(segments)
             bar_line_col.set_zorder((len(ax.get_lines()) - index) + 1)
             break
 
 
 def convert_single_line_to_waterfall(ax, index, x=None, y=None, need_to_update_fill=False):
     line = ax.get_lines()[index]
-
-    amount_to_move_x = index * ax.width * (ax.waterfall_x_offset / 500) if x is None else \
```

Comment:
It seems to have been chosen because that is the value that gives the most "natural" looking layout with the default values, I chose 100 for a similar reason.

Reply:
In that case it seems more transparent for the original non-log code to use default % values that are 5 times bigger with the code having factors of 100 in it. That would mean the % indicator in the UI correctly defines what the scaling factors means. Appreciate it's not your changes though so fine to leave if it's me just being pedantic:
![image](https://user-images.githubusercontent.com/56295817/175039277-a3fe0000-a43b-4fc5-9357-7f896368b866.png)